src/app/components/journey/journey.component.ts

```diff
@@ -1,54 +1,17 @@
-import {Component, OnInit} from '@angular/core';
+import {Component, Input} from '@angular/core';
 import {Journey} from '../../models/Journey';
-import gql from 'graphql-tag';
-import {map, shareReplay} from 'rxjs/operators';
-import {Apollo} from 'apollo-angular';
-import {Observable} from 'rxjs';
 
 @Component({
   selector: 'app-journey',
   templateUrl: './journey.component.html',
   styleUrls: ['./journey.component.scss']
```

Comment:
What is this component?

Reply:
this is a component to display a single journey